README.md

```diff
@@ -12,8 +12,14 @@ Repository to describe, develop, document, and test the service APIs for Dedicat
 ## Scope
 
 * Service APIs for DedicatedNetworks (see [APIBacklog.md](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md))
-* The API provides the customer with the ability to:  
-  * TDB: §Short API scope description§ 
+* The API provides the customer with the ability to:
+  * request/modify/delete a (logical) dedicated network for API consumers. 
+  * A dedicated network offers
+    * specific connectivity performance (e.g. speed/ latency) 
+    * during a specific time duration (e.g. one hour) and
+    * at specific locations (e.g. the area of a festival)
+  * A dedicated network may be ensured by an SLA.
+  * API consumers is be able to handle access to this dedicated network for devices.
```

Comment:
API consumers are able to manage access to this dedicated network for specific devices.

Reply:
Updated. Thanks for the improvement